src/seed.ts

```
@@ -0,0 +1,247 @@
+          data: {
+            name: category.name,
+            slug: category.slug,
+            color: category.color || undefined,
+            parent: null,
+          },
+        });
+        createdParents.push(parentCategory);
+        console.log(`Created parent category: ${category.name}`);
+      } catch (error) {
+        console.error(`Error creating parent category ${category.name}:`, error.data?.errors || error.message);
+        continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)